scripts/zones/Apollyon/IDs.lua

```diff
@@ -18,16 +18,22 @@ zones[xi.zone.APOLLYON] =
         LOGIN_CAMPAIGN_UNDERWAY       = 7002, -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!
         LOGIN_NUMBER                  = 7003, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
         MEMBERS_LEVELS_ARE_RESTRICTED = 7023, -- Your party is unable to participate because certain members' levels are restricted.
-        TIME_EXCEEDED                 = 7065, -- You have exceeded the time limit. Exiting Limbus now.
-        CONDITION_FOR_LIMBUS          = 7066, -- You have clearance to enter Limbus, but cannot enter while you or a party member is engaged in battle.
+        TIME_IN_THE_BATTLEFIELD_IS_UP = 7065, -- Your time in the battlefield is up! Now exiting...
+        PARTY_MEMBERS_ARE_ENGAGED     = 7066, -- You have clearance to enter Limbus, but cannot enter while you or a party member is engaged in battle.
         HUM                           = 7078, -- You hear a faint hum.
+        NO_BATTLEFIELD_ENTRY          = 7083, -- There are several six-sided indentations and a thin slot in the center of the circular dais here.
         NO_KEY                        = 7083, -- There are several six-sided indentations and a thin slot in the center of the circular dais here.
-        PARTY_WILL_BE_REMOVED_KICK    = 7358, -- All party members in Limbus have fallen in battle. Exiting now.
         THE_PARTY_WILL_BE_REMOVED     = 7366, -- All party members in Limbus have fallen in battle. Exiting in <number> [minute/minutes].
+        MEMBERS_OF_YOUR_PARTY         = 7332, -- Currently, <amount> members of your party (including yourself) have clearance to enter Limbus.
+        MEMBERS_OF_YOUR_ALLIANCE      = 7333, -- Currently, <amount> members of your alliance (including yourself) have clearance to enter Limbus.
+        TIME_LIMIT_FOR_THIS_BATTLE_IS = 7335, -- You may stay in Limbus for <amount> minutes.
+        PARTY_MEMBERS_HAVE_FALLEN     = 7358, -- All party members in Limbus have fallen in battle. Exiting now.
+        YOU_INSERT_THE_CARD_POLISHED  = 7371, -- You insert the <card> polished with <cosmo cleanse> into the slot!
         CHIP_TRADE                    = 7372, -- The light in the <item> has grown dim.
-        TIME_EXTENDED                 = 7374, -- our time in Limbus has been extended <number> [minute/minutes].
```

Comment:
Despite it looking wrong, update extractor actually needs this truncated
I believe the reason is that sometimes the first character of the message has bits for color and corrupts the character, so it's skipped. This means if there is an ID shift update extractor will not update `TIME_EXTENDED` and `TIME_LEFT` correctly

You can check these yourself by running the [update extractor ](https://github.com/zach2good/UpdateExtractor) yourself and looking at the output at `out\scripts\zones\Apollyon\Text.lua.raw.txt`

there might be a way for us to mask out the color bits so this isn't a problem in the future

Reply:
Ah ok, thanks for explaining that to me. I undid that "fix" I made so I don't break those.